cpp/src/io/parquet/reader_impl_chunking_utils.cu

```diff
@@ -80,9 +80,9 @@ void print_cumulative_page_info(device_span<PageInfo const> d_pages,
   }
 }
 
-void print_cumulative_row_info(host_span<cumulative_page_info const> sizes,
-                               std::string const& label,
-                               std::optional<std::vector<row_range>> splits = std::nullopt)
+void print_cumulative_page_info(host_span<cumulative_page_info const> sizes,
```

Comment:
Renamed this as it prints stuff from `cumulative_page_info` instead of (unused and now removed) `cumulative_row_info` struct.